### ISSUE TITLE
`hint -;` should add hint labels only on 'actually-scrollable' elements

### ISSUE THREAD

**Issue by bew** [tridactyl/tridactyl#4502]
Looking at https://gitlab.com/explore/projects/trending , then opening `What's New` sidebar (found in the question mark menu on top right), now I wanted to scroll in the `What's New` sidebar.

So this is the screen:
![Screenshot_20221211_215249](https://user-images.githubusercontent.com/9730330/206928360-4f482106-420b-487b-80e6-f7dab48db6b2.png)

When I do `;;` I get MAANNYYYY hints, and it hard to find the correct one, even though there are basically only 2 scrollable elements on this page... (the main page and the sidebar)
![Screenshot_20221211_215232](https://user-images.githubusercontent.com/9730330/206928388-9540e348-37de-4c09-8be8-65f2c37f445f.png)

Quick search lead me to: https://stackoverflow.com/a/29956778
Maybe this could be used to decide if an element is scrollable, thus if there should be a hint label for it?

-   Tridactyl version: `1.23.0`
-   Firefox version: `105.0.3`

**Comment by bovine3dom** [comment]
I agree, we should offer a hint mode that is only for picking what to scroll. The trouble is that I hacked it on top of the focus hint mode which can be used for e.g. bringing up the context menu on links, so we can't remove all the elements from `;;`.

**Comment by treapster** [comment]
doing `bind ;; hint -;` dramatically reduces the number of hints.

**Comment by bovine3dom** [comment]
Yeah, but you're gambling that there's always going to be a link in the part you want to scroll. We can probably craft a better selector. I think surfingkeys or vimfx can do "which element to scroll" so maybe we should look at their code
